<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="combo_mode">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>311</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Results Only</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Frame Skip</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Frame-by-Frame</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>291</y>
      <width>68</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>7</y>
      <width>261</width>
      <height>285</height>
     </rect>
    </property>
    <property name="title">
     <string>Parameters</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>168</x>
       <y>61</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Beta Error</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>61</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Db</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>61</y>
       <width>16</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>s</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_Da">
     <property name="geometry">
      <rect>
       <x>78</x>
       <y>111</y>
       <width>19</width>
       <height>160</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="slider_Db">
     <property name="geometry">
      <rect>
       <x>148</x>
       <y>111</y>
       <width>19</width>
       <height>160</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>6250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>61</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Da</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_s">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>111</y>
       <width>19</width>
       <height>160</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>0</number>
     </property>
     <property name="value">
      <number>3125</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="slider_betaError">
     <property name="geometry">
      <rect>
       <x>198</x>
       <y>111</y>
       <width>19</width>
       <height>160</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="text_s">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>81</y>
       <width>51</width>
       <height>26</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="plainText">
      <string>12</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="text_Da">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>81</y>
       <width>51</width>
       <height>26</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="text_Db">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>81</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="text_BetaError">
     <property name="geometry">
      <rect>
       <x>188</x>
       <y>81</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>29</y>
       <width>68</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Preset:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_presets">
     <property name="geometry">
      <rect>
       <x>74</x>
       <y>24</y>
       <width>169</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Figure 2A - Large Spots</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Figure 2B - Small Spots</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Figure 2C - Cheetah</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Figure 2D - Leopard</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Figure 3A - Irregular Spots</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Figure 3B - Reticulation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Figure 3C - Random Stripes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Figure 3D - Mixed Large and Small Stripes</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_draw">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>311</y>
      <width>98</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Draw</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_out">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>58</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_frameSkip">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>344</y>
      <width>132</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_next">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>344</y>
      <width>98</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Next</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_frame">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>34</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame: 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
